tools/docker_run_ci

```diff
@@ -8,5 +8,5 @@ docker images | grep opensuse
 
 docker run --env "QEMU_QMP_CONNECT_ATTEMPTS=10" --env "EXPECTED_ISOTOVIDEO_RUNTIME=20" \
   $CI_ENV --rm --entrypoint '' -v "$PWD":/opt $IMAGE \
-  sh -c 'cd /opt && ./tools/install-new-deps.sh && ./autogen.sh && make && make check coverage-codecov VERBOSE=1'
+  sh -c 'cd /opt && ./tools/install-new-deps.sh && cmake -DCMAKE_BUILD_TYPE=Release . && make -j4 check WITH_COVER_OPTIONS=1 && make coverage-codecov VERBOSE=1'
```

Comment:
The `-j4` is hardcoded for something that can change. Also we should try to just have the need for one make target

Reply:
Not sure whether nproc is available within that context but I can try.

It was even using three make targets before so I already consider this an improvement. Of course it would be nice having to use only `coverage-codecov`. However, that target only runs the tests it needs to run in order to compute the coverage and not e.g. the spell checking or install tests. (Of course I could add a target to combine check and coverage-codecov but I don't think that makes it easier to understand what we're doing within the CI.)

Reply:
I've switched to ninja which executes a reasonable number of jobs in parallel by default.